new/LIT7126Asmat.xml

```diff
@@ -0,0 +1,78 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7126Asmat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1"><foreign xml:lang="gez">ʾAsmāt</foreign>-names of God found by St Michael</title>
```

Comment:
As above: 

I prefer to use either "ʾAsmāt of Michael" or "Names of Michael", because ʾasmāt literally means 'names'.

I am not sure about the St before Michael. I think for the most common saints (i.e. Mary, archangels), we do not write St in the title.